File: Computabilidade-e-Complexidade-de-Algoritmos-main/golang/12.go
package main
import "fmt"
import "strings"

func contem110(s string) bool {
    return strings.Contains(s, "110")
}

func main() {
    fmt.Println(contem110("1110"))  // True
    fmt.Println(contem110("1011"))  // False
}

File: Computabilidade-e-Complexidade-de-Algoritmos-main/java/12.java
public class AFNContem110 {
    public static boolean contem110(String s) {
        return s.contains("110");
    }

    public static void main(String[] args) {
        System.out.println(contem110("1110"));  // True
        System.out.println(contem110("1011"));  // False
    }
}
